### PR TITLE
added edit for range question

### PR DESCRIPTION
**Pull request by DavidHaighton**

### CODE REVIEW

**Comment by DavidHaighton** [comment]
ignore branch name lmao

File: src/main/java/com/based/basedsurvey/controller/WebQuestionController.java [hunk 1]
```diff
@@ -16,7 +16,8 @@
 @Log
 @Controller
 public class WebQuestionController {
-    private final String MULTIPLE_CHOICE_SHORT = "MQC";
+    private final String MULTIPLE_CHOICE_SHORT = "MQC",
+    OPEN_ANSWER_QUESTION = "OAQ";
```
Comment (Shashaank5) [inline src/main/java/com/based/basedsurvey/controller/WebQuestionController.java:20]: it might make sense to use an enum here

Comment (DavidHaighton) [reply src/main/java/com/based/basedsurvey/controller/WebQuestionController.java:20]: thymeleaf makes it really hard :(
i know https://www.baeldung.com/thymeleaf-enums exists, but it didn't work for me
I think it'd be more effort than its worth

**Review by Shashaank5 — APPROVED** [review]
Based

**Pull request merged**